_episodes/16-parallel.md

```diff
@@ -553,8 +553,8 @@ if rank == 0:
 ```
 {: .language-python}
 
-A fully commented version of the final MPI parallel python code is available
-[here](/files/pi-mpi.py).
+A fully commented version of the final MPI parallel python code is available:
+[pi-mpi.py]({{ site.url }}{{ site.baseurl }}/files/pi-mpi.py).
```

Comment:
From [what I've read](https://mademistakes.com/mastering-jekyll/site-url-baseurl/), `{{ site.url }}` is not necessary...but it also does no harm so merging this